### PR TITLE
Add changelog

### PR DESCRIPTION
**Pull request by VicDeo**
Closes https://github.com/owncloud/files_antivirus/issues/173

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+
+## [0.10.1.0] - Unreleased
+
+### Changed 
+
+- DB schema ported from xml to migrations
```
Comment (PVince81) [inline CHANGELOG.md]: can we have the PR numbers + links ?

**Comment by VicDeo** [comment]
@PVince81 updated

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,226 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+### Changed
```
Comment (PVince81) [inline CHANGELOG.md]: stray "Changed"

**Comment by VicDeo** [comment]
@PVince81 removed

**Comment by PVince81** [comment]
@vicdeo please backport to stable10

**Comment by VicDeo** [comment]
Stable10: https://github.com/owncloud/files_antivirus/pull/185

**Pull request merged**